tests/netperf_test.py

```diff
@@ -13,35 +13,40 @@
 import time
 import os
 from devices import board, wan, lan, wlan, prompt
-
 def install_netperf(device):
     # Check version
     device.sendline('\nnetperf -V')
     try:
-        device.expect('Netperf version 2.4', timeout=10)
+        device.expect('Netperf version 2.6', timeout=10)
         device.expect(device.prompt)
     except:
         # Install Netperf
         device.sendline('apt-get update')
-        device.expect(device.prompt, timeout=60)
-        device.sendline('apt-get -o DPkg::Options::="--force-confnew" -y --force-yes install netperf')
-        device.expect(device.prompt, timeout=60)
-    device.sendline('/etc/init.d/netperf restart')
-    device.expect('Restarting')
-    device.expect(device.prompt)
+        device.expect(device.prompt, timeout=120)
+        device.sendline('apt-get  install netperf')
+        device.expect(device.prompt, timeout=120)
 
 class NetperfTest(rootfs_boot.RootFSBootTest):
-    '''Setup Netperf and Ran Throughput.'''
+    '''Setup Netperf and Run Throughput.'''
+    iface_dut = "eth1"
```

Comment:
Ideally we use this on the devices we are using.

lan.iface_dut
wan.iface_dut

This is not an item that is test specific but device specific so we can extract them from those places.